### ISSUE TITLE
[BUG] Inline code blocks are rendered incorrectly

### ISSUE THREAD

**Issue by cserteGT3** [tlienart/JuDoc.jl#198]
Example from `index.md` of a freshly generated site:
```
### LaTeX and Maths

Essentially three things are imitated from LaTeX

1. you can introduce definitions using `\newcommand`
1. you can use hyper-references with `\eqref`, `\cite`, ...
1. you can show nice maths (via KaTeX)
```
should be rendered like:
### LaTeX and Maths

Essentially three things are imitated from LaTeX

1. you can introduce definitions using `\newcommand`
1. you can use hyper-references with `\eqref`, `\cite`, ...
1. you can show nice maths (via KaTeX)

but looks like the following (note that this is edited by hand):
### LaTeX and Maths

Essentially three things are imitated from LaTeX

1. you can introduce definitions using
`newcommand`
1. you can use hyper-references with
`eqref`, 
`cite`, ...
1. you can show nice maths (via KaTeX)

This happens on `#master`, but not on `v0.2.0`.

**Comment by tlienart** [comment]
Thanks for this, it's another one of those where `<p>` are inserted and should not. I'll investigate and fix asap

**Comment by tlienart** [comment]
Ok this is now fixed, the issue is described in #205 and solved in #209 ; on master the following:

```md
Essentially three things are imitated from LaTeX
1. you can introduce definitions using `\newcommand`
1. you can use hyper-references with `\eqref`, `\cite`, ...
1. you can show nice maths (via KaTeX)
```

becomes

```html
sentially three things are imitated from LaTeX</p>
<ol>
    <li><p>you can introduce definitions using <code>\newcommand</code></p></li>
    <li><p>you can use hyper-references with <code>\eqref</code>, <code>\cite</code>, ...</p></li>
    <li><p>you can show nice maths &#40;via KaTeX&#41;</p></li>
</ol>
```

So there's no spurious insertions of `<br/>` that caused the line skipping you reported.

Thanks for the report!
